lib/SyncAsyncFileSystemDecorator.js

```diff
@@ -16,6 +16,16 @@ function SyncAsyncFileSystemDecorator(fs) {
 			}
 			callback(null, result);
 		};
+
+		this.statSync = function(arg) {
```

Comment:
This is unneccessarily complex. You can write it as:

``` js
this.statSync = function(arg) {
  return fs.statSync(arg);
}
```

for all of them

Reply:
Okay, i will make those changes